assets/css/styles.scss

```
@@ -13,3 +13,13 @@
 #content h1:hover .anchor::before {
 	content: unset;
 }
+
+.home-emoji {
+	font-size: 40px;
```

```suggestion
	font-size: 32px;
```

i'd go for slightly smaller emojis:
- this matches better the delta-chat-logo (so that emojis are not bigger than that).
- i'd go for the nicer 💬 instead of 🗨️  - and 💬 is larger on most platforms anyway.

---

Done